scripts/microbe_stage/organelle.as

```diff
@@ -190,7 +190,7 @@ class Organelle{
         }
 
         offset /= count;
-        return offset;
+        return ((offset/=count))*HEX_SIZE;
```

Comment:
This does not make sense to me. The function is named "calculateCenterOffset", which it should do instead of dividing the average again.

Reply:
i also multiply by hex size after

Reply:
Copying my suggestion from discord:

make a new method called soemthing like `calculateOffsetForOrganelleModel` which calls calculateCenterOffset and then applies this extra math on top.

Reply:
fixed